AboutPage.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <title>raji harthi|Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div class="inline">
+                <div class="inline"> <a  class="current" href="AboutPage.html">About me</a> |</div>
+                <div class="inline">  <a href="Portfolio.html">Portfolio</a></div>
+              
+            </div>
+        </nav>
+   
+        
+        <h1>raji harthi|</h1>
+       <hr width=70%>
+    
+        <main>
+        <div class="clearfix">
+                <img class="img2" src="/Users/RayG/Desktop/sei/lesson/lesson-w02d01-html-css/imges/userlogo.png" alt="raji harthi" width="170" height="170">
```

Comment:
image link not working. Should be "starter-code/images/userlogo.png" as the image is in the starter code folder.